@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -552,7 +552,7 @@ export class VisibleTileSet {
                 // Update the visible area of the tile. This is used for those tiles that are
                 // currently loaded and are waiting to be decoded to sort the jobs by area.
                 tile.visibleArea = tileEntry.area;
-                tile.setElevation(tileEntry.minElevation, tileEntry.maxElevation);
+                tile.elevationRange = tileEntry;
```

Comment:
I guess you missed this within last PR update.

Reply:
i don't understand what you expect here now. I'm copying the elevation range inside the setter, what's the problem then?

Reply:
Oh... ok, I did not spotted that.